src/uwtools/drivers/chgres_cube.py

```diff
@@ -33,8 +34,11 @@ def __init__(
         :param config: Path to config file (read stdin if missing or None).
         :param dry_run: Run in dry-run mode?
         :param batch: Run component via the batch system?
+        :param key_path: Path of keys to subsection of config file.
```

Comment:
I wonder if this might benefit from a little more specificity: It's not just _any_ subsection of the config we're looking for (and it's not necessarily a file -- it may be data streamed on `stdin` -- so we should probably not use that word here), it's the driver-configuration block. Maybe something like
```
:param key_path: Keys leading through the config to the driver's configuration block.
```
? Here and in the other driver modules?